README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+We use the following icons for our organization and repository logos:
+- [Gear icon](https://thenounproject.com/search/?q=gear&i=102547) by Jardson Almeida, from the Noun Project
```

Comment:
Please add a new line between a paragraph and a list for consistency in our Markdown =/